test/domain.test.ts

```diff
@@ -80,6 +129,16 @@ describe('store: [adapter=DOMAIN]', () => {
 
       globalStore.initialize(adapter)(callback);
     });
+
+    it('should return null if invoked under active domain w/o proper store initialization.', done => {
+      const d = domain.create();
+
+      d.run(() => {
+        // Ensure data in the existing domain is available at this point.
+        expect(globalStore.getAll()).to.equal(null);
+        done();
+      });
+    });
```

Comment:
Duplicate test case. This is the same as the one above.

Reply:
The previous one was done for `get()` this one is for `getAll()` they are identical tests but for different functions.

Reply:
Didn't realize it.